torch/nn/quantized/modules/activation.py

```diff
@@ -183,3 +183,53 @@ def from_float(mod):
     @classmethod
     def from_reference(cls, mod, scale, zero_point):
         return cls(mod.dim, float(scale), int(zero_point))
+
+class PReLU(torch.nn.Module):
+    r"""This is the quantized equivalent of :class:`~torch.nn.PReLU`.
+
+    Args:
+        scale: quantization scale of the output tensor
+        zero_point: quantization zero point of the output tensor
+        num_parameters: number of parameters: 1, or the number of channels at input. Default: 1
+    """
+    def __init__(self, output_scale: float, output_zero_point: int,
+                 num_parameters: int = 1) -> None:
+        super(PReLU, self).__init__()
```

Comment:
nit: `super().__init__()`

Reply:
Hi @jerryzh168 I have rebased the PR and fixed this issue. But for the `prelu_quantized_cpu` you mentioned above, I am not sure how to reuse that. It needs additional arguments. Could you give some suggestions? Thanks!

Reply:
Hi @jerryzh168 This PR is merged. If you think reuse of `_test_activation_impl` is still a problem, please let me know. Thanks.